assets/newsletter-section.css

```
@@ -2,6 +2,49 @@
   padding-right: calc(4rem / var(--font-body-scale));
   padding-left: calc(4rem / var(--font-body-scale));
 }
+.newsletter__wrapper-new {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 5rem;
```

```suggestion
  gap: 6rem;
```
Just feels a bit more accurate.